__tests__/app.test.js

```diff
@@ -29,4 +31,21 @@ describe(('GET/api/topics'), () => {
               ])
         })
     })
-})
\ No newline at end of file
+})
+describe("GET/api", () => {
+  test('200: responds with a full list of created endpoints as a JSON object', () => {
+    const expectedEndpoints = fs.readFile(`${__dirname}/../endpoints.json`, 'utf-8')
```

Comment:
can just require in the json file instead. This will remove need for promise.all.